video_xblock/static/js/video_xblock.js

```diff
@@ -21,6 +26,21 @@ function VideoXBlockStudentViewInit(runtime, element) {
     });
   }
 
+  /** Save video player analytic event by POSTing it to VideoXBlock handler */
+  function publishEvent(eventHandlerUrl, data) {
```

Comment:
can we avoid duplications by reusing `saveState` function (probably we will need to rename it)?